mappings/net/minecraft/util/registry/RegistryKey.mapping

```diff
@@ -1,16 +1,45 @@
 CLASS net/minecraft/class_5321 net/minecraft/util/registry/RegistryKey
+	COMMENT Represents a key for a value in a registry in a static context.
+	COMMENT
+	COMMENT @param <T> the type of the value
 	FIELD field_25136 INSTANCES Ljava/util/Map;
-	FIELD field_25137 registryId Lnet/minecraft/class_2960;
-	FIELD field_25138 valueId Lnet/minecraft/class_2960;
+		COMMENT A cache of all registry keys ever created.
```

Comment:
*Why, Mojang?*

Reply:
Fyi this is a map, not some sort of guava loading cache. I don't know why but they apparently probably wants it.